### PR TITLE
add missing 'wheel' extensions to Spark 2.4.0 easyconfig using intel/2018b toolchain

### PR DESCRIPTION
**Pull request by boegel**
Installation fails now due to auto-downloaded dependency being detected (because `download_dep_fail` is set via `exts_default_options`).

When this easyconfig was added it went undetected because of a bug (which was fixed in https://github.com/easybuilders/easybuild-easyblocks/pull/1530)...

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3112.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/7fdb451274b3fa84335422676d174b36 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2611.swalot.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fcbae3c8b72c34c8288724c8b2582278 for a full test report.

File: easybuild/easyconfigs/s/Spark/Spark-2.4.0-intel-2018b-Hadoop-2.7-Java-1.8-Python-3.6.6.eb [hunk 1]
```diff
@@ -31,9 +31,13 @@ exts_default_options = {
 }
 
 exts_list = [
+    ('wheel', '0.33.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/w/wheel'],
```
Comment (akesandgren) [inline easybuild/easyconfigs/s/Spark/Spark-2.4.0-intel-2018b-Hadoop-2.7-Java-1.8-Python-3.6.6.eb]: Add 'source_urls': [PYPI_SOURCE] to exts_default_options and drop source_urls?

Comment (verdurin) [reply easybuild/easyconfigs/s/Spark/Spark-2.4.0-intel-2018b-Hadoop-2.7-Java-1.8-Python-3.6.6.eb]: @boegel Any update on this to address Ake's comments?

Comment (boegel) [reply easybuild/easyconfigs/s/Spark/Spark-2.4.0-intel-2018b-Hadoop-2.7-Java-1.8-Python-3.6.6.eb]: fixed in f29f7abc6e

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/5f5008859aa8670516302cc638a440f8 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**